TestFoundation/TestNSHTTPCookieStorage.swift

```diff
@@ -241,9 +241,14 @@ class TestNSHTTPCookieStorage: XCTestCase {
         let bundlePath = bundle.bundlePath
         let pathIndex = bundlePath.range(of: "/", options: .backwards)?.lowerBound
         let task = Process()
-        task.launchPath = bundlePath.substring(to: pathIndex!) + "/xdgTestHelper/xdgTestHelper"
+        #if os(macOS)
```

Comment:
This should really just be using the Bundle API directly to find the correct path in the bundle, which would be platform independent.